atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -118,6 +123,33 @@ public void create(byte[] tableMetadata) {
                 "INSERT INTO " + config.metadataTable().getQualifiedName() + " (table_name, table_size) VALUES (?, ?)");
     }
 
+    private void maybeAlterTableToHaveOverflow() {
+        try {
+            String shortTableName = oracleTableNameGetter.getInternalShortTableName(conns, tableRef);
+
+            if (config.alterTablesOrMetadataToMatch().contains(tableRef)
```

Comment:
Open question: tableRefs are case sensitive - do we want operators to get the casing right (but this is more specific, and safer, but harder for them to get right), or is a case insensitive match better here? :)

Reply:
Good question! I don't feel strongly either way, but seeing as the table reference is type sensitive, it's probably good that we case match just to ensure the right table is altered.